app/helpers/alchemy/pages_helper.rb

```diff
@@ -286,105 +281,24 @@ def render_page_title(options = {})
-    end
-
-    # Renders a html <meta> tag for name: "" and content: ""
-    #
-    # === Webdevelopers:
-    #
-    # Please use the render_meta_data() helper. There all important meta information gets rendered in one helper.
-    # So you dont have to worry about anything.
-    #
-    def render_meta_tag(options = {})
-      default_options = {
-        name: "",
-        default_language: "de",
-        content: ""
-      }
-      options = default_options.merge(options)
-      lang = (@page.language.blank? ? options[:default_language] : @page.language.code)
-      tag(:meta, name: options[:name], content: options[:content], lang: lang)
-    end
-
-    # This helper takes care of all important meta tags for your page.
-    #
-    # The meta data is been taken from the @page.title, @page.meta_description, @page.meta_keywords, @page.updated_at and @page.language database entries managed by the Alchemy user via the Alchemy cockpit.
-    #
-    # Assume that the user has entered following data into the Alchemy cockpit of the Page "home" and that the user wants that the searchengine (aka. google) robot should index the page and should follow all links on this page:
-    #
-    # Title = Homepage
-    # Description = Your page description
-    # Keywords: cms, ruby, rubyonrails, rails, software, development, html, javascript, ajax
-    #
-    # Then placing +render_meta_data(title_prefix: "Company", title_separator: "-")+ into the <head> part of the +pages.html.erb+ layout produces:
-    #
-    #   <meta charset="utf-8">
-    #   <title>Company - #{@page.title}</title>
-    #   <meta name="description" content="Your page description">
-    #   <meta name="keywords" content="cms, ruby, rubyonrails, rails, software, development, html, javascript, ajax">
-    #   <meta name="created" content="Tue Dec 16 10:21:26 +0100 2008">
-    #   <meta name="robots" content="index, follow">
-    #
-    def render_meta_data(options = {})
-      if @page.blank?
-        warning("No Page found!")
-        return nil
-      end
-      default_options = {
-        title_prefix: "",
-        title_separator: "",
-        default_lang: "de"
-      }
-      options = default_options.merge(options)
-      # render meta description of the root page from language if the current meta description is empty
-      if @page.meta_description.blank?
-        description = Language.current_root_page.try(:meta_description)
+    def meta_description
+      if @page.meta_description.present?
+        @page.meta_description
       else
-        description = @page.meta_description
+        Language.current_root_page.try(:meta_description)
       end
-      # render meta keywords of the root page from language if the current meta keywords is empty
-      if @page.meta_keywords.blank?
-        keywords = Language.current_root_page.try(:meta_keywords)
+    end
+
+    def meta_keywords
+      if @page.meta_keywords.present?
+        @page.meta_keywords
```

Comment:
`presence` respects that. Please have a look into the documentation http://api.rubyonrails.org/classes/Object.html#method-i-presence